dashboard/test/lib/services/lti_test.rb

```diff
@@ -68,4 +69,13 @@ class Services::LtiTest < ActiveSupport::TestCase
     student_user = Services::Lti.initialize_lti_user(@id_token)
     assert_equal student_user.name, @id_token[@custom_claims_key][:given_name]
   end
+
+  test 'create_lti_user_identity should create an LtiUserIdentity when given LTI User' do
+    @id_token[@roles_key] = ['not-a-teacher-role']
+    user = Services::Lti.initialize_lti_user(@id_token)
+    user.age = 15
+    user.save
```

Comment:
Do we need to call save here? If so, do we then need to perma-delete the user from the db to prevent this from filling up our local dbs?

Reply:
You're right, this is left over from an abandoned angle I was taking. We don't need to call save here since we are just testing the `initialize_lti_user` method. I was also testing the `after_create` callback here, but ended up moving that to `user_test.rb`.

Reply:
Actually, digging into this, I don't think db entries created in tests persist long term. Calling `User.all` in my local rails console doesn't show any user entities that were created by the tests.

Reply:
I know there's a separate `test` db locally, but I'm not sure how to connect to it - maybe it gets wiped after each test run?

Reply:
Yeah, you're right. ChatGPT says the test db gets cleared on it's own. I'll dig into it, but if you look at the user_test.rb, almost none of those entities are being cleaned up explicitly in the code.